apis/nucleus/src/components/listbox/ListBoxPopoverWrapper.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import ListBoxPopover from './ListBoxPopover';
+
+export default function ListBoxPopoverWrapper({
+  app,
+  fieldIdentifier,
+  stateName,
+  element,
+  anchorOrigin,
+  transformOrigin,
+  options = {},
+}) {
+  const [showState, setShowstate] = useState(!!options.show);
+  const handleCloseShowState = () => {
+    setShowstate(false);
+  };
+
+  return (
+    <ListBoxPopover
+      show={showState}
+      app={app}
+      alignTo={{ current: element }}
```

Comment:
Is this some ref trick? why not just pass `element` as it is?

Reply:
reason is `<ListboxPopover />` component has been used in nebula itself. I would assume that in internal use case of nebula , for `alignTo` prop, then entire ref object has been passed so we are already using it as `alignTo.current` inside of `<ListboxPopover />`component.


**on the other hand:** when we expose popover function -> we expect to receive an actual element as an anchor point, and not a ref (because not every environment has to be react to have a ref object, it could be mashup for example ). 

so we accept the overhead of wrapping actual element into ref format for internal usage like `{ current: element }`
(or pls correct me if I'm wrong in this case)

Reply:
looks like a ref trick yes, but I think I just copied this from somewhere... but now I can't find it..